docs/src/runtime/programs.md

```diff
@@ -64,21 +64,21 @@ Deploys, upgrades, and executes programs on the chain.
 - Program id: `BPFLoaderUpgradeab1e11111111111111111111111`
 - Instructions: [LoaderInstruction](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/loader_upgradeable_instruction/enum.UpgradeableLoaderInstruction.html)
 
-The BPF Upgradeable Loader marks itself as "owner" of the executable and
+The BPF Upgradeable Loader marks itself as the "owner" of the executable and
 program-data accounts it creates to store your program. When a user invokes an
-instruction via a program id, the Solana runtime will load both your the program
+instruction via a program id, the Solana runtime loads both your program
 and its owner, the BPF Upgradeable Loader. The runtime then passes your program
-to the BPF Upgradeable Loader to process the instruction.
+to the BPF Upgradeable Loader for it to process the instruction.
 
 [More information about deployment](../cli/examples/deploy-a-program.md)
 
 ## Ed25519 Program
 
-Verify ed25519 signature program. This program takes an ed25519 signature, public key, and message.
+The program for verifying ed25519 signatures. It takes an ed25519 signature, a public key, and a message.
 Multiple signatures can be verified. If any of the signatures fail to verify, an error is returned.
 
 - Program id: `Ed25519SigVerify111111111111111111111111111`
-- Instructions: [new_ed25519_instruction](https://github.com/solana-labs/solana/blob/master/sdk/src/ed25519_instruction.rs#L45)
+- Instructions: [ed25519_instruction](https://docs.rs/solana-sdk/1.18.18/solana_sdk/ed25519_instruction/index.html)
```

Comment:
To properly link to the docs.rs page, let's use the dynamic version rather than a static link
```suggestion
- Instructions: [ed25519_instruction](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/ed25519_instruction/index.html)
```